src/radix-tree/node.ts

```diff
@@ -24,34 +24,34 @@ export function create(end = false, subWord = "", key = ""): Node {
     word: "",
   };
 
-  Object.defineProperty(node, "toJSON", { value: serialize });
-  return node;
+  Object.defineProperty(RadixNode, "toJSON", { value: serialize });
+  return RadixNode;
 }
 
-export function updateParent(node: Node, parent: Node): void {
-  node.parent = parent.id;
-  node.word = parent.word + node.subWord;
+export function updateParent(RadixNode: RadixNode, parent: RadixNode): void {
```

Comment:
Lol. Wrong `Find/Replace All`, isn't it? I think you want to restore the variables names here XD

Reply:
shoot 😂